servers/audio/audio_rb_resampler.cpp

```diff
@@ -102,16 +102,16 @@ uint32_t AudioRBResampler::_resample(AudioFrame *p_dest, int p_todo, int32_t p_i
 
 			float v0 = rb[(pos * 6) + 0];
 			float v1 = rb[(pos * 6) + 1];
-			float v2 = rb[(pos * 6) + 2];
-			float v3 = rb[(pos * 6) + 3];
-			float v4 = rb[(pos * 6) + 4];
-			float v5 = rb[(pos * 6) + 5];
-			float v0n = rb[(pos_next * 6) + 0];
-			float v1n = rb[(pos_next * 6) + 1];
-			float v2n = rb[(pos_next * 6) + 2];
-			float v3n = rb[(pos_next * 6) + 3];
-			float v4n = rb[(pos_next * 6) + 4];
-			float v5n = rb[(pos_next * 6) + 5];
+			//float v2 = rb[(pos * 6) + 2];
```

Comment:
Might be a bug here, shouldn't there be at least:
```
v0 += (v0n - v0) * frac;
v1 += (v1n - v1) * frac;
```
like for `C == 4`? CC @reduz

Reply:
I moved this to an issue too (#20362), I'd suggest to leave the code unmodified for now.